guides/source/action_text_overview.md

```diff
@@ -66,7 +66,7 @@ Then refer to this field in the form for the model:
 <%# app/views/messages/_form.html.erb %>
 <%= form_with(model: message) do |form| %>
   <div class="field">
-    <%= form.label :content %>
+    <%= form.label :title %>
```

Comment:
This is a label for the `content` text area, not a separate input.

I think the point of including `title` alongside `content` in the controller is to show that permitting a rich text attribute works just like permitting a regular attribute.

Reply:
Oh... That's true.
Close the PR.